### PR TITLE
switch tensorflow-gpu module with tensorflow

### PR DESCRIPTION
**Pull request by harshad16**
switch tensorflow-gpu module with tensorflow

## Description
Fixes: #42 

As stated in the issue and on description https://pypi.org/project/tensorflow-gpu/
it is suggested to use `tensorflow` package in place of `tensorflow-gpu`

## How Has This Been Tested?

No changes to be tested.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by atheo89** [comment]
Looks good to me!

File: jupyter/tensorflow/ubi8-python-3.8/Pipfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ name = "pypi"
 [packages]
 # TensorFlow and useful extensions
 
-tensorflow-gpu = "~=2.11.0"
+tensorflow = "~=2.11.0"
```
Comment (VaishnaviHire) [inline jupyter/tensorflow/ubi8-python-3.8/Pipfile:11]: Do we want to make this change in ubi8 images, given that this includes package change ?

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Review by atheo89 — APPROVED** [review]
/lgtm

**Pull request merged**